### PR TITLE
Fix warnings from use of flags in regexes

### LINKED ISSUE #96: Investigate DeprecationWarnings from ply/lex.py
On some Python 3 tests, we're getting warnings that look like the following:
```
ply/lex.py:760: DeprecationWarning: Flags not at the start of the expression '(?P(<t_TAG>'
ply/lex.py:498: DeprecationWarning: Flags not at the start of the expression '(?P(<t_TAG>(?'
```

### PR DESCRIPTION
**Pull request by sydneyli**
fixes #96 ; depends on #95

### CODE REVIEW

File: apacheconfig/parser.py [hunk 1]
```diff
@@ -86,9 +86,9 @@ def reset(self):
             start=self._start,
             outputdir=self._tempdir,
             write_tables=bool(self._tempdir),
-            debug=False,
-            debuglog=log if self._debug else None,
-            errorlog=log if self._debug else None
+            debug=self._debug,
+            debuglog=log if self._debug else yacc.NullLogger(),
+            errorlog=log if self._debug else yacc.NullLogger(),
```
Comment (sydneyli) [inline apacheconfig/parser.py]: This is to suppress ply warnings unless debug is set.

**Comment by etingof** [comment]
Perhaps we should merge #95 prior to this one... I am not sure how GitHub would handle dependent patches in a chain...

**Comment by etingof** [comment]
Thank you!

Is it time for a Christmas release?

**Comment by sydneyli** [comment]
:snowflake: :snowflake: or a new year's one! no rush over the holidays :) :snowflake: :snowflake:

**Pull request merged**